generators/client/templates/react/package.json.ejs

```diff
@@ -26,138 +26,138 @@
     "node_modules"
   ],
   "dependencies": {
-    "@fortawesome/fontawesome-svg-core": "1.2.30",
-    "@fortawesome/free-solid-svg-icons": "5.14.0",
-    "@fortawesome/react-fontawesome": "0.1.12",
-    "availity-reactstrap-validation": "2.7.0",
-    "axios": "0.19.2",
-    "bootstrap": "4.4.1",
+    "@fortawesome/fontawesome-svg-core": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "@fortawesome/free-solid-svg-icons": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "@fortawesome/react-fontawesome": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "availity-reactstrap-validation": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "axios": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "bootstrap": "VERSION_MANAGED_BY_CLIENT_REACT",
     <%_ if (clientTheme !== 'none') { _%>
-    "bootswatch": "4.3.1",
+    "bootswatch": "VERSION_MANAGED_BY_CLIENT_REACT",
     <%_ } _%>
     "dayjs": "VERSION_MANAGED_BY_CLIENT_COMMON",
-    "loaders.css": "0.1.2",
-    "lodash": "4.17.15",
+    "loaders.css": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "lodash": "VERSION_MANAGED_BY_CLIENT_REACT",
     "react": "VERSION_MANAGED_BY_CLIENT_REACT",
     "react-dom": "VERSION_MANAGED_BY_CLIENT_REACT",
-    "react-hot-loader": "4.13.0",
-    "react-jhipster": "0.12.0",
-    "react-loadable": "5.5.0",
-    "react-redux": "7.2.0",
-    "react-redux-loading-bar": "4.6.0",
-    "react-router-dom": "5.1.2",
-    "react-toastify": "5.5.0",
-    "react-transition-group": "4.3.0",
-    "reactstrap": "8.6.0",
-    "redux": "4.0.5",
-    "redux-devtools": "3.5.0",
-    "redux-devtools-dock-monitor": "1.1.3",
-    "redux-devtools-log-monitor": "1.4.0",
-    "redux-promise-middleware": "6.1.2",
-    "redux-thunk": "2.3.0",
+    "react-hot-loader": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "react-jhipster": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "react-loadable": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "react-redux": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "react-redux-loading-bar": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "react-router-dom": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "react-toastify": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "react-transition-group": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "reactstrap": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "redux": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "redux-devtools": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "redux-devtools-dock-monitor": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "redux-devtools-log-monitor": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "redux-promise-middleware": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "redux-thunk": "VERSION_MANAGED_BY_CLIENT_REACT",
     <%_ if (websocket === 'spring-websocket') { _%>
-    "rxjs": "6.4.0",
-    "sockjs-client": "1.3.0",
+    "rxjs": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "sockjs-client": "VERSION_MANAGED_BY_CLIENT_REACT",
```

Comment:
Better to keep dependencies without tests fixed.
There is no test for websocket.

Reply:
ok